server/task/src/main/scala/org/apache/james/task/eventsourcing/WorkerStatusListener.scala

```diff
@@ -24,21 +24,15 @@ import org.apache.james.task.Task.Result
 import org.apache.james.task.eventsourcing.TaskCommand._
 import org.apache.james.task.{TaskId, TaskManagerWorker}
 
-class WorkerStatusListener() extends TaskManagerWorker.Listener {
+case class WorkerStatusListener(eventSourcingSystem: EventSourcingSystem) extends TaskManagerWorker.Listener {
```

Comment:
I guess it's method reorganization. Please split the impossible-to-read changes in a different commit